xbmc/video/dialogs/GUIDialogTeletext.cpp

```diff
@@ -79,6 +79,29 @@ bool CGUIDialogTeletext::OnMessage(CGUIMessage& message)
   return CGUIDialog::OnMessage(message);
 }
 
+void CGUIDialogTeletext::DoProcess(unsigned int currentTime, CDirtyRegionList &dirtyregions)
+{
+  if (m_pTxtTexture)
+  {
+    m_TextDecoder.RenderPage();
```

Comment:
m_TextDecoder.RenderPage() is a rather heavy operation and should only be called if we want render the page.